lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -37,9 +35,6 @@ class HtmlViewEmbedder {
   final Map<int, EmbeddedViewParams> _currentCompositionParams =
       <int, EmbeddedViewParams>{};
 
-  /// The HTML element associated with the given view id.
-  final Map<int?, html.Element> _views = <int?, html.Element>{};
-
   /// The root view in the stack of mutator elements for the view id.
   final Map<int?, html.Element?> _rootViews = <int?, html.Element?>{};
```

Comment:
_rootViews could store a tuple (root, slot) to simply the logic in _compositeWithParams.

Reply:
Added ViewClipChain "tuple" to cache what canvaskit needs to composite a platform view (at least the html part of it)